app/templates/components/paper-autocomplete.hbs

```diff
@@ -34,7 +34,7 @@
       listboxId=(readonly optionsId)
       onFocus=(action "onFocus")
       activate=(action "activate")
-      onBlur=(action "deactivate")
```

Comment:
I couldn't see anything using `deactivate`, but if it's used somewhere I'm sure we can figure out a workaround.  I see we have `activate=(action "activate")` - would adding `deactivate=(action "deactivate")` do the trick here, or do we need to fire both events from `onBlur` for some reason?